xAPI.md

```diff
@@ -507,13 +507,39 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
-## 8.0 Getting More Out of xAPI
 
-### 8.1 Extending xAPI
 
-### 8.2 Profiles
+## 8.0 Extending xAPI
+
+xAPI can be extended in a few ways.  The most notable and extensive is Statement Extensions, which allow great flexibility within Statements.  It is recommended that profiles or Communities or Practice agree on how to use  
+extensions for their particular use cases. Implementation details are covered in a [later section](#miscext)
+
+The About Resource is another place xAPI supports Extensions.  The LRS may find it useful to communicate features or behaviors beyond this specification to activity provider.  It is hoped that if more information is available about an LRS, that the About Resource contains it.
+
+Finally, the set of Resources implemented is not expected to be constrained by this document. Resources beyond 
+those listed in this specification can be implemented and co-exist with current Resources.
+
+<a name="COPs" />
+
+## 9.0 Profiles and Communities of Practice
+
+xAPI focuses on the structure of communication between a client and an LRS.  It spares many details of the content 
+to increase flexibility.  The intention in keeping the specification somewhat loose is that Communities of  
+Practice (CoP) can define content (verbs, activity types, contextual relationships, extensions, etc.) for their use cases.  It is very important that such communities exist and share best practices.
```

Comment:
My proposed wording for this paragraph:

xAPI strictly defines the structure of statements communicated between the client and an LRS but is very flexible as to the content of that statement structure. For example, the specification requires that all statements have a verb property, but does not restrict the values of that property; any verb can be used. This flexibility enables Tin Can to be used in any context, including future use cases not envisaged by the specification authors.

It is intended that Communities of Practice (CoP) will define the content of statements (verbs, activity types, contextual relationships, extensions, etc.) to be used in their use cases. It is very important that such communities exist and share best practices.